xpdschema/schemas/run_start.json

```diff
@@ -0,0 +1,178 @@
+    },
+    "sp_plan_name":{
+      "type": "string",
+      "description": "name of ScanPlan"
+    },
+    "sp_args":{
+      "type": "array",
+      "description": "positional arguments passed to ScanPlan"
+    },
+    "sp_kwargs":{
+      "type": "object",
+      "description": "keyword arguments passed to ScanPlan"
+    },
+    "sp_uid":{
+      "type": "string",
+      "description": "auto-generated uid for ScanPlan object"
+    },
+    "sp_time_per_frame":{
+      "type": "number",
+      "description": "acquisition time per frame in second"
+    },
+    "sp_num_frames":{
+      "type": "number",
+      "description": "number of frames being summed per trigger"
+    },
+    "sp_requested_exposure":{
+      "type": "number",
+      "description": "user requested exposure time in second"
+    },
+    "sp_computed_exposure":{
+      "type": "number",
+      "description": "program computed exposure time"
+    },
+    "sp_type":{
+      "type": "string",
+      "description": "ScanPlan type of xpdacq"
+    },
+    "sp_startingT":{
+      "type": "number",
+      "description": "starting temperature of xpdAcq Tramp ScanPlan"
+    },
+    "sp_endingT":{
+      "type": "number",
+      "description": "ending temperature of xpdAcq Tramp ScanPlan"
+    },
+    "sp_requested_Tstep":{
+      "type": "number",
+      "description": "user requested temperature step of xpdAcq Tramp ScanPlan"
+    },
+    "sp_computed_Tstep":{
+      "type": "number",
+      "description": "program computed temperature step of xpdAcq Tramp ScanPlan"
+    },
+    "sp_Nsteps":{
+      "type": "number",
+      "description": "program computed number of steps of xpdAcq Tramp ScanPlan"
+    },
+    "calibration_md":{
+      "type": "object",
+      "description": "dictionary of calibration parameters linked by xpdAcq, Depreciation Warning: this will depreciated in upcoming releases, please do not use this going forward"
+    },
+    "mask":{
+      "type": "array",
+      "description": "sparse representation of the mask generated by auto-mask function, Depreciation Warning: this will depreciated in upcoming releases, please do not use this going forward"
+    },
+    "energy_calibration_client_uid":{
+      "type": "string",
+      "description": "Key linking to energy calibration"
+    },
+    "energy_calibration_server_uid":{
+      "type": "string",
+      "description": "Key linking to energy calibration"
+    },
+    "detector_calibration_client_uid":{
+      "type": "string",
+      "description": "uid of calibration collection"
+    },
+    "detector_calibration_server_uid":{
+      "type": "string",
+      "description": "uid of calibration collection"
+    },
+    "background_client_uid":{
+      "type": "string",
+      "description": "uid of background"
+    },
+    "background_server_uid":{
+      "type": "string",
+      "description": "uid of background"
+    },
+    "dark_client_uid":{
+      "type": "string",
+      "description": "uid of dark"
```

Comment:
"Key linking to the right dark-count server. See discussion in the Notes section above